pages/team.tsx

```diff
@@ -32,6 +33,22 @@ export default function Team({ members, error }: Props) {
         </Heading>
       </Flex>
 
+      <Grid
+        templateColumns={{
```

Comment:
sei que chakra tem também um `gridTemplateColumns`. Sabe qual a diferença entre ele e o `templeateColumns`?

Se não, da uma pesquisada e conta pra gente se achar algo. To curioso, pois sempre uso o primeiro, mas não sei exatamente a diferença.

Reply:
Pelo que vi aqui `templateColuns` é  uma forma condensada de escrever para o `gridTemplateColumns`. Segundo a doc  do Chakra, a `templateColumns` só funciona se utilizada  no componente Grid.

Reply:
Aí fiquei com uma dúvida,  isso significa que a princípio pode-se usar `gridTemplateColuns` dentro de um componente Box?

Reply:
Sim sim, esse `gridTemplateColuns` pode ser usado em qualquer elemento, mas não é recomendável, senão não teria um componente próprio pra grid né?

mas so pra explicar, o `gridTemplateColuns` é o nativo do HTML saca, por isso bateu a duvida se esse ai funcionava. Mas sendo só um atalho, faz sentido.